recipes/sharedarray/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "SharedArray" %}
+{% set version = "3.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7a47d3d2aacf07428438e938dee609d2306859ff5e2ed20df7fd28bc6bea3db
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+  run:
+    - python
+    - numpy
```

Comment:
Should be:
```
  - {{ pin_compatible('numpy') }}
```


https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy